### ISSUE TITLE
Improve screen reader support for Unsupported Block

### ISSUE THREAD

**Issue by etoledom** [wordpress-mobile/gutenberg-mobile#950]
Screen reader should be able to select this block (currently it's not selectable).
It should read something like:
"Unsupported block. {Block name}". (i.e: Unsupported block. Gallery).
